docs/source/api-reference/array-manipulation-routines.rst

```diff
@@ -1,18 +1,14 @@
-# Array Manipulation Routines
-
-.. currentmodule:: nums.numpy
-
-
-Array manipulation routines
+Array Manipulation Routines
 ***************************
 
-.. currentmodule:: numpy
+.. currentmodule:: nums.numpy
 
 Basic operations
 ================
 .. autosummary::
    :toctree: generated/
 
+    copyto
```

Comment:
This isn't hyperlinked when I compile docs?

Reply:
It links if you merge #551

Reply:
Gotcha, thanks for clarifying